client/src/commons/CustomSwal/useCustomSwal.ts

```diff
@@ -4,19 +4,21 @@ import useStyles from './CustomSwalStyles';
 const useCustomSwal = () => {
     const classes = useStyles();
 
-    const alert = (title: string,icon: SweetAlertIcon, options?: SweetAlertOptions) =>
+    const alert = (title: string, text: string, icon: SweetAlertIcon, options?: SweetAlertOptions) =>
         Swal.fire({
             title,
+            text,
             icon,
             customClass: {
-                title: classes.swalTitle
+                title: classes.swalTitle,
+                content: classes.swalText
             },
             ...options
         });
 
-    const alertError = (title: string, options?: SweetAlertOptions) => alert(title, 'error', options);
-    const alertWarning = (title: string, options?: SweetAlertOptions) => alert(title, 'warning', options);
-    const alertSuccess = (title: string, options?: SweetAlertOptions) => alert(title, 'success', options);
+    const alertError = (title: string, text?: string,  options?: SweetAlertOptions) => alert(title, text ? text : '', 'error', options);
```

Comment:
maybe pass `text` inside options instead of adding another property? 
then components that only use title and options would have to pass undefined like `alertError(title, undefined, options)`

Reply:
i mean usage would be like this (w/ existing useCustomSwal)
```
   alertError(
                'לא הצלחנו לחשב את מורכבות החקירה מחדש',
                {text : 'שים לב לשדות סטטוסים, גורם מאבטח ותחום עיסוק'},
            );
```